algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -113,7 +116,8 @@ def model_fn(
 
     with contexts[mode]():
       inputs, input_paddings = augmented_and_preprocessed_input_batch['inputs']
-      logits, logits_paddings = model(inputs, input_paddings)
+      logits, logits_paddings = model(inputs.to(DEVICE),
```

Comment:
do we need to put inputs and input_paddings to device everytime model_fn is called ?

Reply:
Yes!